vaadin-button.html

```diff
@@ -17,30 +17,39 @@
 -->
 
 <link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vaadin-input/vaadin-control-state-mixin.html">
 
 <dom-module id="vaadin-button">
   <template>
     <style>
       :host {
-        display: block;
+        display: inline-block;
+        position: relative;
+      }
+
+      button {
+        width: 100%;
+        height: 100%;
       }
     </style>
+    <button id="button"
+        role="button"
```

Comment:
This is most likely redundant. The native button element should already be announced as a button. You don’t need this when using the button element on a normal website, so why would you need it inside shadow DOM?